tests/python/test_clear_all_gradients.py

```diff
@@ -2,6 +2,7 @@
 
 
 @ti.test()
```

Comment:
```suggestion
```
You don't have to repeat `ti.test()` :-)

Reply:
oops nice catch! thanks!